modules/data-streams/src/main/java/org/elasticsearch/datastreams/action/PromoteDataStreamTransportAction.java

```diff
@@ -31,9 +35,14 @@
 import org.elasticsearch.threadpool.ThreadPool;
 import org.elasticsearch.transport.TransportService;
 
+import static org.elasticsearch.core.Strings.format;
+
 public class PromoteDataStreamTransportAction extends AcknowledgedTransportMasterNodeAction<PromoteDataStreamAction.Request> {
 
+    private static final Logger logger = LogManager.getLogger(PromoteDataStreamTransportAction.class);
+
     private final SystemIndices systemIndices;
+    private final ClusterService clusterService;
```

Comment:
I think this can also be removed, right?

Reply:
Yep. Removed :-)